src/Be.Vlaanderen.Basisregisters.GrAr.Contracts/Common/AddressRegistryReaddress.cs

```diff
@@ -0,0 +1,17 @@
+namespace Be.Vlaanderen.Basisregisters.GrAr.Contracts.Common
+{
+    public sealed class AddressRegistryReaddress
```

Comment:
@ArneD we hebben deze verplaatst naar Common. Is in principe een breaking change ...